packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -0,0 +1,103 @@
+<div class="ds-c-usa-banner">
+  <header class="ds-c-usa-banner__header">
+    <svg
+      width="16"
+      height="11"
+      class="ds-c-usa-banner__header-flag"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
One thing you can do similar to other html examples using svg is reference the svg file directly using `<use href> </use>`. See the button icons html example